ui/apps/platform/src/Containers/Vulnerabilities/VirtualMachineCves/aggregateUtils.ts

```diff
@@ -0,0 +1,26 @@
+import type { VirtualMachine } from 'services/VirtualMachineService';
+import type { CVSSV3Severity } from 'types/vulnerability.proto';
+
+// Most if not all functions in this file will be removed once backend filtering is implemented.
```

Comment:
Adding a `@TODO` might help with searching for comments like this when we need to revisit them.